bitnami/thanos/templates/ruler/statefulset.yaml

```diff
@@ -86,6 +86,7 @@ spec:
           args:
             - rule
             - --log.level={{ .Values.ruler.logLevel }}
+            - --log.format={{ .Values.bucketweb.logFormat }}
```

Comment:
I think there is a copy& paste error here, shouldn't this be `.Values.ruler.logFormat` ?

If it is, please review the other places where `logFormat` is used.

Reply:
Indeed, I'm caught red handed :)
I'll double check other values.

Reply:
Fixed